xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-test/xwiki-platform-localization-test-docker/src/test/it/org/xwiki/localization/test/ui/TranslationsRestIT.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.localization.test.ui;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.xwiki.localization.rest.TranslationsResource;
+import org.xwiki.test.docker.junit5.TestReference;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Functional tests of the translations REST endpoint.
+ *
+ * @version $Id$
+ * @since 13.3RC1
+ */
+@UITest
+class TranslationsRestIT
+{
+    @BeforeEach
+    void setUp(TestUtils testUtils)
+    {
+        testUtils.loginAsSuperAdmin();
+    }
+
+    @Test
+    @Order(1)
+    void translationsNoKeyParam(TestUtils testUtils) throws Exception
+    {
+        Map<String, Object[]> queryParams = new HashMap<>();
+        GetMethod getMethod = testUtils.rest().executeGet(TranslationsResource.class, queryParams, "xwiki");
+        String body = getMethod.getResponseBodyAsString();
+        assertEquals("<Translations><translation/></Translations>", body);
```

Comment:
I don't understand where the **upper case** ``Translations`` comes from but I doesn't look right to me. In your schema you have:

    <element name="translations" type="localization:Translations"/>

as root element, so the name should be "translations" not "Translations". For instance the REST URL ``rest/wikis/xwiki/spaces/Sandbox/pages/WebHome/attachments`` outputs ``<attachments><attachment>...`` and we have:

    <element name="attachments" type="xwiki:Attachments"></element>

in the [schema](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-model/src/main/resources/xwiki.rest.model.xsd#L435)